assets/src/scss/layout/_navbar.scss

```diff
@@ -129,11 +129,6 @@ $navbar-default-height: 60px;
         display: flex;
       }
 
-      .nav-search-toggle-container {
-        cursor: unset;
-        pointer-events: none;
```

Comment:
On medium width, menu opened, clicking on the search toggle doesn't close the menu/remove the overlay if you remove the pointer-events rule. (In a previous version it was also opening the search bar while keeping the overlay)

Reply:
Right, with the current structure that's indeed needed. I guess ideally we would add the overlay over everything except the opened menu, but I'll change it back for now.

Reply:
The overlay is over everything outside of nav-main, might be easier to have a different search toggle outside of the nav-main for medium and smaller widths